translate.py

```diff
@@ -64,13 +34,10 @@ def is_valid_gast(gast):
 
 
 def gast_to_code_caller(gast, output_lang, error_handler):
-    # try:
     output_code = gtc.gast_to_code(gast, output_lang)
 
+    # TODO: remove last static reference to language
```

Comment:
@cjoshea9 can you resolve this? Will prob require adding a field to the converter

Reply:
[Taiga#207](https://tree.taiga.io/project/jackdavidweber-cjs-capstone/us/207?no-milestone=1)